### PR TITLE
Help bottom sheet

### LINKED ISSUE #712: Info bubbles to explain
## Goal: reduce confusion

We can solve common users questions with strategically placed info bubbles / "learn more" links.

## Details

Common confusion points

- **Deposit screen**
  - From B: “What would you recommend as easiest way to move into Daimo…”?
  - “Transferring from CB vs from a wallet I already own“ … “how do I decide?”
- **Account screen**
   - Why USDC? / is this a Coinbase app?
   - Where is my seed phrase? / how do passkeys work?
- **Transfer screen**
   - How did that work without me paying gas?

## Proposed fix

(?) bubble at various points

These can open to a full-page in-app bottomsheet concise explainer.

In some cases, (Learn more) button can link out to blog post or web page.

### PR DESCRIPTION
**Pull request by wojtus7**
Kinda Closes #712 (3/4)

Add new help modal and use it for three new modals (Onboarding, Settings and Transfers)

Transfer Confirmations help looks like change to the history bottom sheet so I rather have it as separate PR probably together with #830

Android:

https://github.com/daimo-eth/daimo/assets/42337257/23cd84d6-82bf-464f-a4d7-75a4316e42df

https://github.com/daimo-eth/daimo/assets/42337257/52fec762-ea3c-4f85-9404-d13c5922f7e2

iOS:

https://github.com/daimo-eth/daimo/assets/42337257/a3a1d174-26eb-4546-bb0e-6914e2a0ef4f

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/SettingsScreen.tsx [hunk 1]
```diff
@@ -211,6 +214,21 @@ function DevicesSection({ account }: { account: Account }) {
         title="Create a Passkey Backup"
         message="Secured by your password manager"
         icon={<ClockIcon color={color.gray3} style={{ top: 7 }} />}
+        onPressInfo={() =>
+          dispatcher.dispatch({
+            name: "helpModal",
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/SettingsScreen.tsx:219]: factor this out into a function or constant

File: apps/daimo-mobile/src/view/screen/SettingsScreen.tsx [hunk 2]
```diff
@@ -229,16 +247,30 @@ function ButtonInfo({
...
 }) {
   return (
     <View style={styles.buttonInfoContainer}>
       {icon}
       <View style={styles.messageContainer}>
-        <TextBody color={color.midnight}>{title}</TextBody>
+        <View style={styles.textRow}>
+          <TextBody color={color.midnight}>{title}</TextBody>
+          {onPressInfo && (
+            <TouchableOpacity onPress={onPressInfo}>
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/SettingsScreen.tsx:264]: 👍

File: apps/daimo-mobile/src/view/screen/onboarding/OnboardingIntroScreen.tsx [hunk 3]
```diff
@@ -67,6 +69,59 @@ export function OnboardingIntroScreen() {
...
+        Daimo stores money via cryptographic secrets. There's no bank.
+      </IntroTextParagraph>
+    </IntroPage>,
+    <IntroPage title="On Ethereum">
+      <IntroTextParagraph>
+        Daimo runs on Base, an Ethereum rollup. This lets you send money
+        securely, anywhere in the world.
+      </IntroTextParagraph>
+    </IntroPage>,
+  ];
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/onboarding/OnboardingIntroScreen.tsx:123]: 👍

File: apps/daimo-mobile/src/view/shared/AmountInput.tsx [hunk 4]
```diff
@@ -42,8 +46,35 @@ export function AmountChooser({
...
+          <TextLight>
+            Rollups support near-instant transactions that cost a few cents each
+            by bundling many transactions into a single L1 transaction. They
+            inherit the strong guarantees of Ethereum: like L1, Base is reliable
+            and secure, and works worldwide.
+          </TextLight>
+          <Spacer h={24} />
+          <TouchableOpacity
+            onPress={() => {
+              // TODO add link
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/shared/AmountInput.tsx:70]: https://l2beat.com/scaling/projects/base

File: apps/daimo-mobile/src/view/shared/AmountInput.tsx [hunk 5]
```diff
@@ -42,8 +46,35 @@ export function AmountChooser({
...
+      content: (
+        <>
+          <TextLight>Daimo uses Base, an Ethereum rollup.</TextLight>
+          <Spacer h={24} />
+          <TextLight>
+            Rollups support near-instant transactions that cost a few cents each
+            by bundling many transactions into a single L1 transaction. They
+            inherit the strong guarantees of Ethereum: like L1, Base is reliable
+            and secure, and works worldwide.
+          </TextLight>
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/shared/AmountInput.tsx:66]: Second paragraph:

| Daimo transactions are sponsored. This means that your transfers are free.

**Review by dcposch — APPROVED** [review]
LGTM after comments

File: apps/daimo-mobile/src/view/shared/AmountInput.tsx (hunk 5 above)
Comment (wojtus7) [reply apps/daimo-mobile/src/view/shared/AmountInput.tsx:69]: I just replaced second paragraph with this 👆

Sorry if you meant something else 🫠

**Comment by wojtus7** [comment]
Ahhhhh sorry I merged master instead of rebase 😳

Soooooorry

**Comment by dcposch** [comment]
All good!

**Pull request merged**